clang/tools/scan-build-py/lib/libear/ear.c

```diff
@@ -503,6 +507,63 @@ int eaccess(const char *pathname, int mode) {
   return call_eaccess(pathname, mode);
```

Comment:
L507  no need call again, just return none zero, as the result is there already.

Reply:
Okay, will refine it in the updated PR.